ui/src/app/models/jobTemplate.model.ts

```diff
@@ -17,10 +17,11 @@ export type JobTemplateModel = {
   id: number;
   name: string;
   formConfig: string;
+  jobType: { name: string };
 };
 
 export class JobTemplateModelFactory {
-  static create(id: number, name: string, formConfig: string): JobTemplateModel {
-    return { id: id, name: name, formConfig: formConfig };
+  static create(id: number, name: string, formConfig: string, jobType: { name: string }): JobTemplateModel {
```

Comment:
There is `JobType` in `app/models/jobInstance.model.ts`

Reply:
Fixed, placed to new file